Dockerfile

```diff
@@ -61,7 +61,8 @@ RUN pip install \
   robotframework-pabot==$PABOT_VERSION \
   robotframework-requests==$REQUESTS_VERSION \
   robotframework-seleniumlibrary==$SELENIUM_LIBRARY_VERSION \
-  robotframework-sshlibrary==$SSH_LIBRARY_VERSION
+  robotframework-sshlibrary==$SSH_LIBRARY_VERSION \
+  PyYAML==5.1
```

Comment:
Two options:
* Either declare the version at the top of the file
* Or don't specify a version for PyYAML (we could rely on the latest version compatible with Python)

I'd go for the latter, as this would have a limited risk to break test cases.